.github/workflows/publish.yaml

```diff
@@ -24,7 +24,12 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - id: package
         run: |
-          PACKAGE_VERSION=$(awk -F '[<>]' '/<Version>/{print $3; exit}' Datadog.Serverless/Datadog.Serverless.Compat.csproj)
+          if [[ ! "$GITHUB_REF" =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
We should probably support prerelease package versions, `x.y.z-tag`, not only `x.y.z`, especially while testing before GA. But feel free to add that in another PR later.

Reply:
I updated the version regex to accommodate any suffixes we may want to add for the released package.

https://github.com/DataDog/datadog-serverless-compat-dotnet/blob/cd11edd3fb1e529f2f5e2a46187e8947aac5f6d9/.github/workflows/publish.yaml#L27